packages/react/src/SelectPanel/SelectPanel.dev.stories.tsx

```diff
@@ -9,13 +9,31 @@ import type {ItemInput} from '../deprecated/ActionList/List'
 import {FeatureFlags} from '../FeatureFlags'
 import FormControl from '../FormControl'
 
-const meta = {
+const meta: Meta<typeof SelectPanel> = {
   title: 'Components/SelectPanel/Dev',
   component: SelectPanel,
 } satisfies Meta<typeof SelectPanel>
 
 export default meta
 
+const emptyMessage = (filter: string) => (
```

Comment:
```suggestion
const EmptyMessage = ({ filter }: { filter: string }) => (
```

I believe this could be a component instead of a helper function, let me know if that's not the case 👀